src/controllers/controllerengine.cpp

```diff
@@ -79,23 +79,23 @@ Input:   -
 Output:  -
 -------- ------------------------------------------------------ */
 void ControllerEngine::callFunctionOnObjects(QList<QString> scriptFunctionPrefixes,
-                                             const QString& function, QScriptValueList args) {
-    const QScriptValue global = m_pEngine->globalObject();
+                                             const QString& function, QJSValueList args) {
+    const QJSValue global = m_pEngine->globalObject();
 
     for (const QString& prefixName : scriptFunctionPrefixes) {
-        QScriptValue prefix = global.property(prefixName);
-        if (!prefix.isValid() || !prefix.isObject()) {
```

Comment:
QScriptValue returns an invalid value if property don't exist. QJSValue returns undefined, thus we can just check isObject().